crates/autopilot/src/driver_model.rs

```diff
@@ -72,13 +72,10 @@ pub mod solve {
     #[derive(Clone, Debug, Serialize)]
     #[serde(rename_all = "camelCase")]
     pub struct Token {
-        pub decimals: Option<u8>,
-        pub symbol: Option<String>,
         pub address: H160,
         #[serde_as(as = "Option<DisplayFromStr>")]
         pub price: Option<U256>,
         #[serde_as(as = "DisplayFromStr")]
```

Comment:
`#[serde_as(as = "DisplayFromStr")]` also needs to get removed to not affect the serialization of `trusted`, I think.